src/utils/utils_log.c

```diff
@@ -66,11 +68,11 @@ typedef struct {
     utils_log_level_t level;
     utils_log_level_t flushLevel;
     FILE *output;
-    const char *file_name;
+    char file_name[MAX_FILE_PATH];
```

Comment:
Hmm, please remember to check for buffer overflow. It seems like a critical security issue. I used to work as security lead at Biedronka.